### PR TITLE
Updated the code styling

### PR DESCRIPTION
**Pull request by icoric**
Changed the code style to look more cleaner and to match the other similar examples

### CODE REVIEW

**Comment by PRMerger12** [comment]
@icoric : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/t-sql/language-elements/bitwise-not-transact-sql.md [hunk 1]
```diff
@@ -63,11 +63,9 @@ monikerRange: ">=aps-pdw-2016||=azuresqldb-current||=azure-sqldw-latest||>=sql-s
  The following example creates a table using the **int** data type to store the values and inserts the two values into one row.  
   
 ```  
-CREATE TABLE bitwise  
-(   
-a_int_value int NOT NULL,  
-b_int_value int NOT NULL  
-);  
+CREATE TABLE #bitwise (
```
Comment (rothja) [inline docs/t-sql/language-elements/bitwise-not-transact-sql.md:66]: Hello. You changed the table to #bitwise but then the insert is into bitwise which will fail. Can you remove the #4880 

```suggestion
CREATE TABLE bitwise (  
```

Comment (icoric) [reply docs/t-sql/language-elements/bitwise-not-transact-sql.md:66]: Sure. I will update it

**Comment by PRMerger17** [comment]
@icoric : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/t-sql/language-elements/bitwise-not-transact-sql.md (hunk 1 above)
Comment (icoric) [reply docs/t-sql/language-elements/bitwise-not-transact-sql.md]: It's updated now

**Comment by rothja** [comment]
@icoric Appreciate the update. Thanks! #sign-off

**Pull request merged**